core/src/main/kotlin/org/sertain/hardware/Joysticks.kt

```diff
@@ -12,3 +12,31 @@ public fun Joystick.whenActive(buttonNumber: Int, command: Command) {
 public fun Joystick.whileActive(buttonNumber: Int, command: Command) {
     JoystickButton(this, buttonNumber).whileActive(command.mirror)
 }
+
+public fun Joystick.whenActive(buttonNumber: Int, block: () -> Unit) {
+    JoystickButton(this, buttonNumber).whenActive(
+        object : edu.wpi.first.wpilibj.command.Command() {
+            override fun execute() {
+                block()
+            }
+
+            override fun isFinished(): Boolean {
+                return true
+            }
```

Comment:
```kotlin
override fun isFinished() = true
```

Reply:
This should be gone, use our command and overloads.

Reply:
How can I use sertain commands here? The `command` parameter in wpilib's whenActive and whileActive expects a wpilib command.

Reply:
@Jamdan2 See `command.mirror` on line 13.

Reply:
Nm i get it